teams/vsp/product-management/product-brief-template.md

```diff
@@ -1,31 +1,36 @@
 # **VSP Product Brief Template**
 
-### What is it?
+To create a Product Brief:
+1. Copy this file to `platform/<product-name>/README.md`.
+1. Fill out all sections below for your product.
+1. Delete this preamble, all italicized instruction blocks, and any unused links within the template.
 
-*Explain how this product will benefit VFS teams.*
+# Product Title
 
-### How does it work?
-
-*Describe at a high level how the product will impact VFS teams' workflow.*
+*Add 1-2 sentences about what this product is and how it will benefit VFS teams.*
```

Comment:
- *Use bullet points for easier visual scanning*
- *Briefly summarize the problem statement and solution*
- *Keep the details in `design-doc.md`*

Reply:
reworded this a bit based on your suggestions. though "briefly summarize problem statement and solution" to me reads more VSP-facing ("what's a statement of the user problem, how are we solving it" --- Product Outline covers this), where this doc is more VFS-facing ("here's what this product is and what it can do for you").